rush.json

```diff
@@ -536,11 +536,6 @@
       "projectFolder": "sdk/eventhub/mock-hub",
       "versionPolicyName": "utility"
     },
-    {
```

Comment:
This hotfix has _nothing_ to do with identity, but this codebase does not contain the necessary changes to support node 16 tests and identity will continue to fail here. I don't think backporting it to support tests for a package I am not releasing (and that search-documents doesn't depend on) is the better approach so I will remove it from the rush.json as we've done in other hotfix branches